### PR TITLE
add paragraph describing independence of sibling subschemas

### LINKED ISSUE #910: Section 9.2.1 (*Of keywords) is non-specific about sequencing
This may be irrelevant to the [test case](https://github.com/json-schema-org/JSON-Schema-Test-Suite/issues/359) that highlighted the issue to me, but it seems that the spec should say whether sequence of evaluation is important.

### PR DESCRIPTION
**Pull request by gregsdennis**

### CODE REVIEW

File: jsonschema-core.xml [hunk 1]
```diff
@@ -2212,6 +2212,13 @@
                     or modifying the subschema results in various ways.
                 </t>
 
+                <t>
+                    Subschemas of these keywords evaluate the instance completely independently
+                    such that the results of one subschema MUST NOT impact the results of sibling
```
Comment (Relequestual) [inline jsonschema-core.xml:2217]:
```suggestion
                    such that the results of one such subschema MUST NOT impact the results of sibling
```

File: jsonschema-core.xml [hunk 2]
```diff
@@ -2212,6 +2212,13 @@
...
+                    subschemas.  The consequence of this is that the subschemas may be applied in
```
Comment (Relequestual) [inline jsonschema-core.xml:2218]:
```suggestion
                    subschemas.  As such, subschemas may be applied in
```

**Comment by Relequestual** [comment]
Made two minor suggestions. Otherwise, I think this is good to go.

**Comment by gregsdennis** [comment]
@Relequestual your suggestions add a lot of redundancy ("such" everywhere).

> Subschemas of these keywords evaluate the instance completely independently **such** that the results of one **such** subschema MUST NOT impact the results of sibling subschemas.  As **such** the subschemas may be applied in any order.

**Comment by Relequestual** [comment]
@gregsdennis I've modified the second such (😆) instance. I just thought your phrasing was a little verbose and potentially not clearly related to the WHOLE previous section.

**Comment by gregsdennis** [comment]
merge when ready

**Review by Relequestual — APPROVED** [review]

**Pull request merged**